app/client/src/ce/reducers/settingsReducer.ts

```
@@ -43,21 +43,21 @@
-    action: ReduxAction<TenantReduxState<any>>,
+    action: ReduxAction<OrganizationReduxState<any>>,
   ) => {
     // TODO: Fix this the next time the file is edited
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     const configs: any = {};
 
-    tenantConfigConnection.forEach((key: string) => {
-      if (action.payload?.tenantConfiguration?.hasOwnProperty(key)) {
-        configs[key] = action.payload?.tenantConfiguration?.[key];
+    organizationConfigConnection.forEach((key: string) => {
+      if (action.payload?.organizationConfiguration?.hasOwnProperty(key)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)